pkg/loadtest/loader.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	KeepRepo         bool
 	Credentials      model.Credential
 	ServerAddress    string
+	NoProgress       bool
```

Comment:
Maybe we can try to avoid the double negative:

```suggestion
	Progress       bool
```

Reply:
Updated the code - just note that I use NoProgress in the first place to have the default `false`.  But for this use-case I don't mind having the default false on show progress.